app/src/main/java/com/infomaniak/mail/ui/newMessage/AiPropositionFragment.kt

```diff
@@ -133,7 +130,11 @@ class AiPropositionFragment : Fragment() {
 
         retryButton.setOnClickListener {
             trackAiWriterEvent("retry")
-            aiViewModel.aiPromptOpeningStatus.value = AiPromptOpeningStatus(isOpened = true)
+            val shouldKeepPrompt = aiViewModel.aiPropositionStatusLiveData.value == PropositionStatus.CONTEXT_TOO_LONG
+            aiViewModel.aiPromptOpeningStatus.value = AiPromptOpeningStatus(
+                isOpened = true,
+                shouldResetPrompt = !shouldKeepPrompt,
+            )
```

Comment:
Why not directly make a variable "shouldResetPrompt" ?

Reply:
I ended up removing the intermediary variable, cause yeah it's not that useful actually